Makefile

```diff
@@ -148,7 +148,7 @@ check:
 clean:
 	$(GO) clean -tags '$(TAGS)' $(GOFLAGS) -i github.com/cockroachdb/...
 	find . -name '*.test' -type f -exec rm -f {} \;
-	rm -rf build/deploy/build
+	rm -rf build/deploy/build .bootstrap
```

Comment:
You're right, will remove that here (and in .dockerignore).

On Mon, Sep 21, 2015 at 7:56 AM, Tamir Duberstein <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/cockroachdb/cockroach/pull/2575#discussion_r39962544>:
>
> > @@ -148,7 +148,7 @@ check:
> >  clean:
> >  	$(GO) clean -tags '$(TAGS)' $(GOFLAGS) -i github.com/cockroachdb/...
> >  	find . -name '*.test' -type f -exec rm -f {} \;
> > -	rm -rf build/deploy/build
> > +	rm -rf build/deploy/build .bootstrap
>
> build/deploy/build is not a thing anymore, is it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2575/files#r39962544>.
>